e2e/README.md

```diff
@@ -119,7 +117,7 @@ ks apply default -c kubeflow-core
 Check to ensure things have deployed:
 
 ```
-$ kubectl logs -l name=tf-job-operator
+$ kubectl logs -l name=tf-job-operator -n ${NAMESPACE}
```

Comment:
I will keep this one and will remove the other namespaces since this is just used once to check  kubeflow is deployed and there is no need to change context!